easybuild/easyconfigs/b/BLAKE3-C/BLAKE3-C-1.6.1-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,38 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2025/03
+easyblock = 'CMakeMake'
+
+name = 'BLAKE3-C'
+version = '1.6.1'
+
+homepage = 'https://github.com/BLAKE3-team/BLAKE3'
+description = """BLAKE3 is a cryptographic hash function that is:
+Much faster than MD5, SHA-1, SHA-2, SHA-3, and BLAKE2.
+Secure, unlike MD5 and SHA-1. And secure against length extension, unlike SHA-2.
+Highly parallelizable across any number of threads and SIMD lanes, because it's 
+a Merkle tree on the inside.
+Capable of verified streaming and incremental updates, again because it's a     
+Merkle tree.
+A PRF, MAC, KDF, and XOF, as well as a regular hash."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://github.com/BLAKE3-team/BLAKE3/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['1f2fbd93790694f1ad66eef26e23c42260a1916927184d78d8395ff1a512d285']
+
+builddependencies = [
+    ('CMake', '3.29.3'),
+    ('binutils', '2.42'),
+]
+dependencies = []
+postinstallcmds = ['cp *.%s %%(installdir)s/lib' % SHLIB_EXT]
+start_dir = 'c'
+configopts = '-DBUILD_SHARED_LIBS=ON '
+configopts += '-DCMAKE_INSTALL_LIBDIR=lib '
+
+sanity_check_paths = {
+    'files': ['lib/libblake3.%s' % SHLIB_EXT],
+    'dirs': ['include/', 'lib/'],
```

Comment:
Drop the /-suffixes on the dir names, and lib is not needed since you have that through files.